packages/vuetify/src/components/VSwitch/VSwitchTrack.tsx

```diff
@@ -0,0 +1,119 @@
+// Styles
+import './VSwitch.sass'
+
+// Components
+import { makeVSelectionControlProps } from '@/components/VSelectionControl/VSelectionControl'
+
+// Composables
+import { useProxiedModel } from '@/composables/proxiedModel'
+
+// Utilities
+import { computed } from 'vue'
+import { genericComponent, propsFactory, useRender } from '@/util'
+
+// Types
+import type { ComputedRef, PropType, Ref } from 'vue'
+import { makeVInputProps } from '../VInput/VInput'
+import type { LoaderSlotProps } from '@/composables/loader'
+import type { GenericProps } from '@/util'
+
+export type VSwitchTrackSlot = {
+  model: Ref<boolean>
```

Comment:
My intent for switch track was to clean-up the component. I don't think that it needs to have model support itself.